lua/net.lua

```diff
@@ -23,9 +24,10 @@ return function (gatekeeper_server)
 	local back_bonding_mode = gatekeeper.c.BONDING_MODE_ROUND_ROBIN
 	local back_vlan_tag = 0x5678
 	local back_vlan_insert = true
+	local back_mtu = 1500
```

Comment:
While 1,500 will be a typical use for the front, 2048 seems more reasonable for the back.